Volume/Supporting/Main.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Main.swift
+//  Volume
+//
+//  Created by Sergio Diaz on 4/26/21.
+//  Copyright © 2021 Cornell AppDev. All rights reserved.
+//
+
+import AppDevAnnouncements
+import Firebase
+import SwiftUI
+
+@main
+struct Main: App {
```

Comment:
.onOpenUrl() would never get called unless I added this @main struct.  To my understanding, this is supposed to be SwiftUI's replacement for Scene/App delegate.  In UIKit, you're supposed to handle opening URL within Scene/App delegate but that doesn't work in SwiftUI because you can't do anything in SwiftUI from Scene/App delegate.